pkg/web/static/openapi.json

```diff
@@ -1275,551 +1292,824 @@
+            "format": "date",
+            "description": "The birthdate of the account holder."
           },
-          "hmac_secret": {
+          "phone": {
             "type": "string",
-            "format": "base64"
+            "description": "The phone number associated with the account."
           },
-          "hmac_algorithm": {
+          "email": {
             "type": "string",
-            "example": "HMAC-SHA-256"
+            "description": "The email address associated with the account."
           },
-          "is_error": {
-            "type": "boolean"
+          "addresses": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "description": "The addresses associated with the account."
           },
-          "error": {
-            "type": "object"
+          "tax_id": {
+            "type": "string",
+            "description": "The tax ID of the account holder."
           },
-          "timestamp": {
+          "language": {
             "type": "string",
-            "format": "datetime"
+            "description": "The preferred language of the account holder."
           },
-          "sealed": {
-            "type": "boolean"
+          "metadata": {
+            "type": "object",
+            "description": "Additional metadata associated with the account."
           },
-          "public_key_signature": {
-            "type": "string"
+          "block_incoming": {
+            "type": "boolean",
+            "description": "Indicates whether incoming transactions are blocked for the account."
           },
-          "original": {
+          "created": {
+            "type": "string",
+            "format": "date-time",
+            "description": "The date and time when the account was created."
+          },
+          "modified": {
             "type": "string",
-            "format": "base64"
+            "format": "date-time",
+            "description": "The date and time when the account was last modified."
           }
         }
       },
-      "DecryptedEnvelope": {
+      "AccountList": {
         "type": "object",
+        "description": "The response object for the accounts endpoint.",
         "properties": {
-          "identity": {
-            "type": "object"
-          },
-          "transaction": {
-            "type": "object"
-          },
-          "pending": {
-            "type": "object"
-          },
-          "sent_at": {
-            "type": "string",
-            "format": "datetime"
+          "page": {
+            "$ref": "#/components/schemas/PageQuery"
           },
-          "received_at": {
-            "type": "string",
-            "format": "datetime"
+          "accounts": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/Account"
+            }
           }
         }
       },
-      "EnvelopeQuery": {
+      "CryptoAddress": {
```

Comment:
The `CryptoAddress` object slightly differs what's listed [here](https://github.com/trisacrypto/envoy/blob/main/pkg/web/api/v1/account.go#L33-L42). Would you mind giving this another look?